components/ProposalStatusBadge.tsx

```diff
@@ -62,13 +74,66 @@ const ProposalStateBadge = ({
   }
 
   return (
-    <div
-      className={`${getProposalStateStyle(
-        proposal.state
-      )} inline-block px-2 py-1 rounded-full text-xs`}
-    >
-      {statusLabel}
-    </div>
+    <>
+      {open ? (
+        <>
+          <div className="flex items-center justify-end gap-4">
+            {governance && proposal.getTimeToVoteEnd(governance) < 0 && (
```

Comment:
It displays Finalise for all Proposals which vote ended. Finalise should only be displayed for proposals in Voting state (see Oyster)